benchmark-framework/.factorypath

```diff
@@ -1,84 +1,139 @@
 <factorypath>
```

Comment:
I don't even really know what this files does, does anyone?

Reply:
I have no idea to be honest... something about annotations in Java? I was just carrying over the commit from the previous PR. I can just drop it.

Reply:
I would drop it, I think it was inadvertently added. Other than that, looks good.